po/ka.po

```diff
@@ -0,0 +1,308 @@
+# A passphrase strength checking and policy enforcement toolset.
+# Copyright (c) 2000-2003,2005,2008,2010,2013,2016 by Solar Designer
+# Copyright (c) 2008,2009 by Dmitry V. Levin
```

Comment:
The copyright lines are expected to match the copyright lines of the whole project,
the corresponding comment in po/passwdqc.pot says:
```
Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
```

By the way, copyright lines in po/ru.po are outdated, too.